ui.apps/src/main/content/jcr_root/apps/core/fd/components/formsportal/searchlister/v1/searchlister/clientlibs/site/css/search.less

```diff
@@ -0,0 +1,91 @@
+/*******************************************************************************
+ * Copyright 2021 Adobe
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ ******************************************************************************/
+
+.cmp-searchlister-search {
+    height: 40px;
+
+    .cmp-searchlister-search__box {
+        position: relative;
+        width: 35%;
+        display: inline-block;
+
+        // search icon via font awesome.
+        &:before {
+            content: "\f002";
+            position: absolute;
+            font-family: 'Font Awesome 5 Free';
```

Comment:
??, is this font approved?

Reply:
I'm not sure. 
I've taken this from https://github.com/adobe/aem-core-wcm-components/blob/61887d1651b517561caaa85486d3b5304b9d8a2f/examples/ui.apps/src/content/jcr_root/apps/core-components-examples/clientlibs/clientlib-site/styles/components/form-text.less#L33